src/viewmodels/event-tiles/TextualEventViewModel.ts

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2025 New Vector Ltd.
+
+SPDX-License-Identifier: AGPL-3.0-only OR GPL-3.0-only OR LicenseRef-Element-Commercial
+Please see LICENSE files in the repository root for full details.
+*/
+
+import { MatrixEventEvent } from "matrix-js-sdk/src/matrix";
+
+import { type EventTileTypeProps } from "../../events/EventTileFactory";
+import { MatrixClientPeg } from "../../MatrixClientPeg";
+import { textForEvent } from "../../TextForEvent";
+import { ViewModelSubscriptions } from "../ViewModelSubscriptions";
+import { type TextualEventViewSnapshot } from "../../shared-components/event-tiles/TextualEvent/TextualEvent";
+
+export class TextualEventViewModel {
```

Comment:
This class should implement https://github.com/element-hq/element-web/pull/30216/files#diff-380541cbe2dd30a3ba936eca60eeac10e0f1f78d10ed65b30c3126b8d1d7a4d2R8 ?